ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-commands.sh

```
@@ -284,6 +284,42 @@ function oc_adm_top_nodes_loop() {
+	do
+		echo "8<----------8<---------- $(date +%s) 8<----------8<----------"
+
+		# Get all disruption-poller pods
+		echo "=== Disruption Poller Pods ==="
+		oc get pods -A | grep -E "disruption-poller|disruption-target" || echo "No disruption pods found"
+
+		echo ""
+		echo "=== Describe Disruption Pods ==="
+		for pod in $(oc get pods -A -o json | jq -r '.items[] | select(.metadata.name | contains("disruption")) | "\(.metadata.namespace)/\(.metadata.name)"'); do
+			echo "--- Pod: $pod ---"
+			oc describe pod -n ${pod%/*} ${pod##*/} 2>&1 | grep -A 30 "Events:" || echo "No events"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)